packages/components/src/components/autocomplete/autocomplete.jsx

```diff
@@ -206,6 +206,8 @@ const Autocomplete = React.memo(props => {
     const onSelectItem = item => {
         if (!item) return;
 
+        if (item?.disabled && props.label === 'Tax residence') return;
```

Comment:
I don't think we can rely on label. it changes in translations. also in components we shouldn't have such a specific check. it should be a new prop etc.

Reply:
Should be fixed